### DISCUSSION TITLE
Cannot seem to go past 1785/921/1600 with Mariko (OLED)

### DISCUSSION THREAD

**Discussion by Kido1986** [hanai3Bi/Switch-OC-Suite#41] [Help | Troubleshooting]
I'm very new to modded Switches so please forgive any ignorance, but I am a fairly technical user in general.

I have modded mariko model and when I load up Sys-clk-oc, I show options for much higher clock speeds on CPU, GPU and MEM, yet when I select them, my clock speeds are limited to 1785/921/1600, seems like they are limited by the 'safe' variables for an Erista model. - https://imgur.com/a/UObXUWe

I have followed installation instructions as carefully as possible. I even found a suggestion to remove libsys_minerva but that did not help.

**Comment by hanai3Bi** [comment]
Read step#5 of how to install

**Reply by Kido1986** [reply]
I have done so but please let m know if I did it improperly:

[config]
autoboot=0
autoboot_list=0
bootwait=0
verification=1
backlight=100
autohosoff=2
autonogc=0
updater2p=1


{--- Custom Firmware ---}
[CFW (SYSMMC)]
fss0=atmosphere/package3
kip1patch=nosigchk
emummc_force_disable=1
icon=bootloader/res/sysnand.bmp
{}

[CFW (EMUMMC)]
kip1=atmosphere/kips/loader.kip
fss0=atmosphere/package3
kip1patch=nosigchk
emummcforce=1
icon=bootloader/res/emummc.bmp
{}




![kips](https://github.com/hanai3Bi/Switch-OC-Suite/assets/21071089/4d2da9ed-f29b-4521-8f5d-cb583ab9aa03)

**Reply by Kido1986** [reply]
So I tried moving the "kip1=atmosphere/kips/loader.kip" line around in the order and if its not the top line, Atmosphere fails to load, blank screen, hard lock.

**Reply by hanai3Bi** [reply]
yes you need to move the location.
if black screen then you did not match OCS version with AMS version.

**Reply by Kido1986** [reply]
Aaaah, Im seeing now, I believe.  I'm on Atmosphere 1.5.4 and your last few versions are only for 1.5.5.  let me see about updating Atmosphere and then trying again. And then I need to relocate the kip1 argument to below the fss line?

Thank you.

**Reply by Kido1986** [reply]
Updating did it!  Thank you so much for pointing out where I made my mistakes! I am learning about this every time I touch it hahah
